.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Not sure you have your .gitignore properly configured?
Shouldn't all the .idea files be ignored, with .gitignore living outside of this directory?